tests/test_attempt_user_login.py

```diff
@@ -74,7 +75,9 @@ def check_user_login_response_valid(response: HTTPResponse) -> bool:
 # fixme: check jwt validity once we get the code to do this
 def check_jwt_valid(response: HTTPResponse) -> bool:
     response_dict = response.json()
-    return "jwt" in response_dict
+    enc_string = response_dict.get('jwt')
+    valid_check = Token.check_if_valid(enc_string)
+    return valid_check
```

Comment:
calling `response_dict.get()` but not handling the default `None` case means that it won't fail on this statement, but instead will fail in the `Token.check_if_valid()` call, which is unintended. 

also, if the `response.json()` _doesn't_ have a `jwt` field in it, then this will fail, instead of returning `False` as it should.